libr/core/cmd_open.c

```diff
@@ -375,6 +375,33 @@ static void cmd_open_bin(RCore *core, const char *input) {
 			}
 		}
 		break;
+	case '=': // "ob="
+		{
+			RListIter *iter;
+			RList *list = r_list_new ();
+			RBinFile *binfile = NULL;
+			RBin *bin = core->bin;
+			const RList *binfiles = bin ? bin->binfiles: NULL;
+			if (!binfiles) {
+				return;
+			}
+			r_list_foreach (binfiles, iter, binfile) {
+				char temp[4];
+				ListInfo *info = R_NEW (ListInfo);
+				if (!info) {
+					return;
+				}
```

Comment:
maybe good to have an r_listinfo_new() to just pass 4 parameters instead of doing the new+struct init by hand

Reply:
Okay :)